deploy/role.yaml

```diff
@@ -84,4 +84,13 @@ rules:
     verbs:
       - list
       - get
-      - watch
\ No newline at end of file
+      - watch
+  - apiGroups:
+      - ""
+    resources:
```

Comment:
isn't the create role missing here?

Reply:
I don't think so, this role is to give only list, get, watch and update in `configmaps` to the `kogito-service-viewer`. This user is attached to the pod, which will update the protobuf `configmap` during container starts.

We could change the name from `kogito-service-viewer` to `kogito-service` though. wdyt?

Reply:
This is how the role appears in the console:
https://i.snipboard.io/WB9Fgi.jpg